### PR TITLE
Adding helper_gui.rst documentation and screenshots

### PR DESCRIPTION
**Pull request by pliu2781**
I added documentation to helper_gui.rst. I put screenshots in another folder, helper_gui_images

### CODE REVIEW

**Comment by sbillinge** [comment]
thanks @pliu2781 this is great.   I will read through and make any suggestions, but please could you share with me an html file of the built docs as I am not currently able to build them.  Don't do it in the PR, just send as an email attachment of sthg.

File: docs/helper_gui.rst [hunk 1]
```diff
@@ -0,0 +1,152 @@
+helper_gui
+==========
+Regolith helpers can add to, list from, or update existing collections in your databases. While helpers can be accessed through command line/terminal, the Helper gui is a more comfortable and accessible way to use them.
+
+As with regular regolith, `helper_gui`  must be run from a directory containing a regolithrc.json file and it will use the config parameters it finds in there.  To use the gui, navigate to such a directory and type `regolith_helper`, e.g.,
```
Comment (sbillinge) [inline docs/helper_gui.rst:5]: you have to type `helper_gui` not `regolith_helper`.    I guess this suggests that you didn't get anyone to test out these insructions.   Please have someone follow the instructions and see how easy it is for them to follow them, then make modifications when you discover that something is not clear.

**Review by sbillinge — COMMENTED** [review]
this looks good, apart from my initial comment which needs a fix, but do try and have someone run through it and see where they have problems.   Then let me know and I can merge it.

**Comment by pliu2781** [comment]
> thanks @pliu2781 this is great. I will read through and make any suggestions, but please could you share with me an html file of the built docs as I am not currently able to build them. Don't do it in the PR, just send as an email attachment of sthg.

Im having trouble with make html. Is there another way to get a html of the helper_gui.rst?

**Pull request merged**